### ISSUE TITLE
Showing geographic  hierarchy on Objects (from 2042)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#89]
**Problem Description:**
Under the Creation section, the "Where" does not display the geographic hierarchy. This can be confusing for users as it will not give context to the location.  

For example, "Suwanee River at Allafhah, below woldtown" can be listed for "Where" but this location does not give context to the user of where it is located. When we click on the location, we are brought to the **place hierarchy** where a user can see that the Suwanee River is located in USA, Florida. This place hierarchy information should be listed within the "where" section. 

**Expected Behavior/Solution:**
Place improvements:
- Add the place hierarchy breadcrumb to "Where"
- The place hierarchy will show under the "where" section as the smallest viewable map tile

**Requirements:**
- The place hierarchy map tiles will show under the "where" section of Objects and Works search results
- The map tiles will be scaled down to the smallest useful size

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Understand how this connects to other Place Hierarchy improvements - Mike, team
- [x] Capture dependencies/blocks - Amy
- [x] Wireframe/Mockup - Mike
- [x] Committee discussions - Sarah
- [x] Feasibility/Team discussion - Sarah
- [x] Backend requirements - TBD
- [x] Frontend requirements- TBD
- [x] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

Place - https://lux-front-tst.collections.yale.edu/view/object/00c21564-de20-427f-b66f-0a109d550594 (tst)
- should include place[ hierarchy](https://lux-front-tst.collections.yale.edu/view/place/47052d52-3945-411c-9db6-99df0e5e9c03)  
- Object Types: https://lux-front-tst.collections.yale.edu/view/object/2563fb25-487b-4d32-af6a-e5a0dbd06e09
- https://lux.collections.yale.edu/view/object/00c21564-de20-427f-b66f-0a109d550594 (prod) 

**Dependencies/Blocks:**

**Related Github Issues:**
https://github.com/project-lux/lux-frontend/issues/152 (taxonomic hierarchy)
**Related links:**
- https://www.bugherd.com/projects/284041/tasks/2053
- https://www.bugherd.com/projects/284041/tasks/1980
- https://www.bugherd.com/projects/284041/tasks/2054
- location in [lux](https://lux-front-tst.collections.yale.edu/view/place/47045dc8-0421-4aec-981a-ba15ac8f120e) 
- Bugherd(Q2/3): https://www.bugherd.com/projects/284041/tasks/2111

**Wireframe/Mockup:**
Current view:
![image](https://github.com/project-lux/lux-frontend/assets/160149818/6a5389cd-2f6f-403f-bcc0-2a4883470cb8)

Expected:
![89](https://github.com/project-lux/lux-frontend/assets/98550395/87acbd69-fe3c-408c-b4ae-2bbadfa0f0e7)

**Comment by prowns** [comment]
Discussed at 3/1 Team meeting:

Mockup updates for @miketullo95 
- Add hierarchy breadcrumb
- Add small map tile with zoom, in place
- Define smallest useful size/scale for tile
- Think about other options if map tile doesn’t work

**Comment by roamye** [comment]
Per UAT | discussion on BH 2054 - this linked hierarchy in specimens is useful. However, unsure whether the hierarchy order is specific to general or general to specific. @prowns will follow up with YPM.

**Comment by prowns** [comment]
@roamye - The taxonomic hierarchy needs more discussion.  Can you split the Object type hierarchy part of this ticket to a new issue, so we can move the geographic/map tile piece forward?

**Comment by roamye** [comment]
@prowns https://github.com/project-lux/lux-frontend/issues/152 split the object type part into this ticket.

**Comment by roamye** [comment]
once mockup is done, we will change to prioritization @miketullo95

**Comment by miketullo95** [comment]
@prowns @roamye mockups uploaded

**Comment by miketullo95** [comment]
@prowns @roamye updated based on feedback. additionally, added some padding between "where" and "additional" per our meeting on Wednesday

**Comment by roamye** [comment]
Looks good in prod, closing.
